tests/test_utils.py

```diff
@@ -51,6 +51,22 @@ def test_get_config(tmpdir):
     }
 
 
+@pytest.mark.parametrize(
+    "verbose", [True, False],
+)
+def test_get_config_log_location(tmpdir, capsys, make_settings, verbose):
+    """Log the location of the .pypirc config used by the user."""
+    pypirc = os.path.join(str(tmpdir), ".pypirc")
+
+    make_settings(verbose=verbose)
+
+    captured = capsys.readouterr()
+    if verbose:
+        assert captured.out == f"Using configuration from {pypirc}\n"
+    else:
+        assert captured.out == ""
```

Comment:
While this test works, I think it's no longer clear why, since `get_config` is a side-effect of `make_settings`. I would probably test for the log message in `test_settings.py`, as a new test after `test_setup_logging`:

https://github.com/pypa/twine/blob/9f76951bcfef646aeb8911de08c84947d8c60205/tests/test_settings.py#L62

Reply:
Sounds good! I made those changes. I'm curious, do you know why `capsys.disabled()` seems to error?

Reply:
>  do you know why capsys.disabled() seems to error?

Nope 😉